HeterogeneousCore/SonicTriton/src/TritonClient.cc

```diff
@@ -325,8 +404,7 @@ void TritonClient::evaluate() {
       reportServerSideStats(stats);
     }
 
-    std::shared_ptr<tc::InferResult> results_ptr(results);
-    success = handle_exception([&]() { getResults(results_ptr); });
+    success = handle_exception([&]() { getResults(results); });

```

Comment:
Would
```suggestion
    success = handle_exception([&]() { getResults(std::move(results)); });
```
work?

Reply:
What is the benefit of moving an `std::vector` in this case?